bucket/reggie-next.json

```
@@ -0,0 +1,24 @@
+{
+    "version": "4.7.0",
+    "description": "Advanced level editor for New Super Mario Bros. Wii originally created by Treeki, Tempus and RoadrunnerWMC using Python, PyQt and Wii.py. Next version created by RoadrunnerWMC, based on official release 3. Milestone 4 version is a collaboration of Horizon users and it aims to add more features requested by users.",
```

Shorten description:

```suggestion
    "description": "Advanced level editor for New Super Mario Bros. Wii",
```